Firebase/Core/Public/FIRTimestamp.h

```diff
@@ -19,41 +19,29 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- * An FSTTimestamp represents an absolute time from the backend at up to nanosecond precision.
- * An FSTTimestamp is represented in terms of UTC and does not have an associated timezone.
+ * An FIRTimestamp represents an absolute time from the backend at up to microsecond precision.
```

Comment:
How about some or all of the description here?

https://github.com/google/protobuf/blob/master/src/google/protobuf/timestamp.proto

Reply:
Does Objective C use the same Javadoc style as Android (i.e., should I modify the link into "@see <a.../>")?

Reply:
Our Swift and Objective-C docs are published using Jazzy which uses a subset of the published standard keywords, documented here:

https://github.com/realm/jazzy#supported-keywords

AFAICT you can't insert the markup around the contents of the `@see`. It's just  `@see URL`, like this:

https://github.com/firebase/firebase-ios-sdk/blob/a9f3f35d483f1031ef2e2860aeda921f56e1bf08/Firebase/Storage/Public/FIRStorageMetadata.h#L28

Reply:
Done.